gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -15,7 +15,7 @@ def self.postgres_dir
     end
 
     def self.postgresql_template
-      RAILS_ROOT.join("../system/TEMPLATE").join(postgres_dir)
+      PostgresAdmin.template_directory.join(postgres_dir)
```

Comment:
It feels like we need a generic template directory on a more generic module/class that these other places can call.